packages/lib/gpc/src/gpcUtil.ts

```diff
@@ -138,21 +139,20 @@ export function canonicalizeSignerPublicKeyConfig(
 export function canonicalizeEntryConfig(
   proofEntryConfig: GPCProofEntryConfig
 ): GPCProofEntryConfig {
+  const canonicalizedBoundsCheckConfig = canonicalizeBoundsCheckConfig({
+    ...(proofEntryConfig.inRange ? { inRange: proofEntryConfig.inRange } : {}),
```

Comment:
This incantation could be simpler if the arg type could accept undefined values.
Or if the two values were passed as two separate arguments, perhaps?